### DISCUSSION TITLE
A Website for Akira

### DISCUSSION THREAD

**Discussion by ghost** [akiraux/Akira#562] [Ideas]
A website that the user can easily download akira from or know more about Akira
## A quick Mock-Up
![Akira Home](https://user-images.githubusercontent.com/77546233/115139745-e6bf3900-a033-11eb-9a15-05a933535846.png)

**Comment by ghost** [comment]
And we can add more content to the site

**Reply by Alecaddd** [reply]
We have a website dedicated repo here: https://github.com/akiraux/website
And I did a mock-up a while ago [here](https://user-images.githubusercontent.com/2527103/115153081-ed947e80-a028-11eb-8278-0d6eb9bd64d3.png)

Until we have a stable release I'm not planning on working on the website to avoid the misconception of Akira being ready to be downloaded and used.

**Reply by ghost** [reply]
I see that you are using vue

**Reply by ghost** [reply]
But no need to use vue you can use basic (HTML, CSS, JS)

**Reply by ghost** [reply]
I can create the website in 1 day with basic web languages

**Reply by abienz** [reply]
I created a fork based on the starter project here https://github.com/abienz/website it still requires a bit of bug fixing, and I think there may have been a requirement to use tailwind css also which was disappointing.

It's kind of been in stasis ever since.

**Reply by ghost** [reply]
I think that we should use basic web languages

**Reply by ghost** [reply]
not Vue

**Reply by abienz** [reply]
Yes I agree, I feel like the Akira maintainers have enough on their plate that they shouldn't worry about the tech stack the website uses and keeping it basic is the best solution.
